splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/joins/NestedLoopJoinOperationIT.java

```diff
@@ -79,6 +135,22 @@ public static void createDataSet() throws Exception {
         createData(spliceClassWatcher.getOrCreateConnection(), schemaWatcher.toString());
     }
 
+    /* Regression test for DB-7673 */
+    @Test
+    public void testIndexLookupWithNestedJoin() throws Exception {
+        ResultSet rs = null;
+        try {
+            rs = methodWatcher.executeQuery("SELECT * FROM v1 --splice-properties useSpark=true\n" +
+                "where M_ID >=1 and M_ID <=100");
+        } catch (SQLException e) {
+            fail("Query not expected to fail.");
+        }
```

Comment:
Catching the exception and invoking `fail` is redundant. You can eliminate the try/catch.